src/FodyNlogAdapter/FodyNlogAdapter.csproj

```diff
@@ -3,9 +3,20 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageId>Stratis.FodyNlogAdapter</PackageId>
-    <Version>3.0.0.4-beta</Version>
     <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
   </PropertyGroup>
+  
+  <PropertyGroup Condition=" '$(Version)' == '' ">
+    <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">3.0.4</VersionPrefix>
```

Comment:
should it not be 3.0.0, rather than 3.0.4?

Reply:
you want to use the build number as the last number so that you can be sure you don't end up with 2 packages having the same number, so on this occasion I have to push that number _4_ one level up